lms/static/js/components/tabbed/views/tabbed_view.js

```diff
@@ -41,13 +66,21 @@
                        var self = this;
                        this.$el.html(this.template({}));
                        _.each(this.tabs, function(tabInfo, index) {
-                           var tabEl = $(_.template(tabTemplate, {
-                               index: index,
-                               title: tabInfo.title,
-                               url: tabInfo.url
-                           }));
+                           var tabId = tabInfo.url,
+                               tabEl = $(_.template(tabTemplate, {
+                                   index: index,
+                                   title: tabInfo.title,
+                                   url: tabInfo.url,
+                                   tabPanelId: tabId
```

Comment:
Tabs contain a `aria-controls` attribute which identifies the tab panel that is owned by that tab. The value of `aria-controls` should be the `id` of the panel, hence I wanted to pass in an explicit `tabPanelId`. It's more of an implementation detail that this happens to be the same as `tabInfo.url`. That being said, it looks like there's no reason for the variable `tabId` to exist, so I'll get rid of it.